### ISSUE TITLE
Feature Request: Support for BiblioCraft Atlas in Camping GUI Map Slot

### ISSUE THREAD

**Issue by PitchBright** [Rikmuld/CampingMod-mcmod#66]
It would be great to be able to use the BiblioCraft Atlas (collection of maps) in the Camping GUI Map Slot.

**Comment by mando222** [comment]
This would help a pack I am working on as well.

**Comment by Rutes** [comment]
The Antique Atlas would be great too!

**Comment by jonqrandom** [comment]
yeah, compat with the BiblioCraft atlas would be a perfect addition.

**Comment by Rikmuld** [comment]
Sure, I'll try to add it in the next version.
